### PR TITLE
[Minor] Created Swagger for Account, Project, and Auth endpoints.

### LINKED ISSUE #187: Create Swagger for Account, Project and Auth endpoints
Epic/User story #123

### PR DESCRIPTION
**Pull request by Manraj45**
closes #187

### CODE REVIEW

**Review by jmezaber — APPROVED** [review]
Nice!

File: backend/swagger/cirrusspark.yaml [hunk 1]
```diff
@@ -0,0 +1,1257 @@
...
+  description: "API for managing tasks"
+- name: "auth"
+  description: "API for authorization"
+- name: "pet"
+  description: "Everything about your Pets"
+  externalDocs:
+    description: "Find out more"
+    url: "http://swagger.io"
+- name: "store"
+  description: "Access to Petstore orders"
```
Comment (davidthai0387) [inline backend/swagger/cirrusspark.yaml:33]: is this part of the template you used?

File: backend/swagger/cirrusspark.yaml [hunk 2]
```diff
@@ -0,0 +1,1257 @@
...
+        "404":
+          description: "Not found"
+        "401":
+          description: "Unauthorized"
+        "500":
+          description: "Unexpected error"
+        "200":
+          description: "OK"
+  /accounts/allEmployees:
+    post:
```
Comment (davidthai0387) [inline backend/swagger/cirrusspark.yaml:98]: should be GET here i think

File: backend/swagger/cirrusspark.yaml [hunk 3]
```diff
@@ -0,0 +1,1257 @@
...
+      - "application/json"
+      produces:
+      - "application/json"
+      responses:
+        "404":
+          description: "Not found"
+        "200":
+          description: "OK"
+  /accounts/allEmployees/{businessId}:
+    post:
```
Comment (davidthai0387) [inline backend/swagger/cirrusspark.yaml:114]: here as well

**Review by davidthai0387 — CHANGES_REQUESTED** [review]
There's a few 'pet' routes that should be deleted if they were part of the template you used. Otherwise great work!

File: backend/swagger/cirrusspark.yaml (hunk 1 above)
Comment (Manraj45) [reply backend/swagger/cirrusspark.yaml:33]: Yes, we kept the pet endpoint as an example. We can remove it once we are done creating the Swagger

File: backend/swagger/cirrusspark.yaml (hunk 3 above)
Comment (Manraj45) [reply backend/swagger/cirrusspark.yaml:114]: Yes you're right. I merged with Simon's YAML so he might have made some mistakes

**Review by davidthai0387 — APPROVED** [review]
Looks good

**Pull request merged**